wasmcloud_host/lib/wasmcloud_host_web/live/components/actor_row_component.ex

```diff
@@ -36,13 +36,7 @@ defmodule ActorRowComponent do
         <button id="copy_actor_id_<%= @actor %>_<%= @host_id %>" class="btn btn-sm btn-primary" data-toggle="tooltip"
           data-placement="top" title data-original-title="Copy Actor ID"
           onClick="navigator.clipboard.writeText('<%= @actor %>')">
-          <svg class="c-icon" style="color: white">
-            <use xlink:href="/coreui/free.svg#cil-copy"></use>
-          </svg>
-        </button>
-        <button id="copy_host_id_<%= @actor%>_<%= @host_id %>" class="btn btn-sm btn-info" data-toggle="tooltip"
-          data-placement="top" onClick="navigator.clipboard.writeText('<%= @host_id %>')" title
-          data-original-title="Copy Host ID">
+          <%= String.slice(@actor, 0..4) %>&#46;&#46;&#46;
```

Comment:
Can we replace the three `&#46`'s with one single `&#8230`? That's the HTML entity for an ellipses character

Reply:
Also, if you could add `id-monospace` into the list of CSS classes on the button it would be consistently styled with the link row and host info box buttons

Reply:
Updated.